build/Nuget/Microsoft.NET.Sdk.Nuget.targets

```diff
@@ -192,11 +103,37 @@
 
       <CopyLocalBuildFile Include="@(CopyLocalBuildCrossTargetingFile)" />
       <CopyLocalBuildFile Include="@(CopyLocalSdkFile)" />
+
+      <!-- Use the contents of the output directory of the net46 build of the SDK tasks, except
+           for MSBuild and Roslyn DLLs and their dependencies-->
+      <Net46OutputAssembly Include="$(OutDir)net46\*.dll" />
```

Comment:
Do you have to update the authoring in https://github.com/dotnet/sdk/blob/master/src/VsixV3/Microsoft.Net.Sdk.swr? Did the layout change?